package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "img-magnifier-extension",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dhruvdhaduk-simform/img-magnifier-extension.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/dhruvdhaduk-simform/img-magnifier-extension/issues"
+  },
+  "homepage": "https://github.com/dhruvdhaduk-simform/img-magnifier-extension#readme",
+  "description": "",
```

Can add a suitable description:
```suggestion
  "description": "A browser extension for magnifying images.",
```

---

Added description.
https://github.com/dhruvdhaduk-simform/img-magnifier-extension/pull/1/commits/564cf393c8f6b659c85d8609710d7e9c55c2c0c0